component/commons/handler/src/main/java/org/wso2/carbon/identity/password/expiry/validation/handler/PasswordExpiryPolicyUtils.java

```diff
@@ -0,0 +1,67 @@
+package org.wso2.carbon.identity.password.expiry.validation.handler;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.IdentityProvider;
+import org.wso2.carbon.identity.application.common.model.IdentityProviderProperty;
+import org.wso2.carbon.identity.application.common.util.IdentityApplicationManagementUtil;
+import org.wso2.carbon.identity.event.IdentityEventConfigBuilder;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.bean.ModuleConfiguration;
+import org.wso2.carbon.idp.mgt.IdentityProviderManagementException;
+import org.wso2.carbon.idp.mgt.IdentityProviderManager;
+
+/**
```

Comment:
A class method should say about a class in general. Not about the methods it contains. We should say what a util class is intended to do.